_src/pattern-library/sass/utilities/_variables.scss

```diff
@@ -331,6 +332,46 @@ $buttons-elevated-shadow: 3px !default;
 $buttons-border-radius: 3px !default;
 $buttons-font-weight: font-weight(semi-bold) !default;
 
+// forms
+$label-color: $text-base-color;
```

Comment:
Please make sure all of these variables, which should be customizable, have the Sass ``!default`` flag, so that they can be overridden later in [the theming workflow](https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#compiling-a-theme-on-top-of-an-applications-sass).